src/lib/index.js

```diff
@@ -727,6 +722,8 @@ lib.isIOS = function() {
     return IS_IOS_REGEX.test(window.navigator.userAgent);
 };
 
+// Do we need this process now that IE9 and IE10 are not supported?
```

Comment:
yeah, we could probably get rid of this now - and revert the change here that's its only consumer: https://github.com/plotly/plotly.js/commit/98adfaee3ed79c6f23a7fb8bbe1fe27474229f23#diff-9c325f1fe845cf45c705407012f4d4d0be31a9d9fbead70d1f70c5dedde9c634

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/5373/commits/42c97e2127384de010e7c3de3c4d42369f03087a.